components/Navbar/Login.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Login = () => {
+  return (
+    <div>
```

Comment:
Don't make this a component directly embed a button in navbar.tsx without div wrapper